### PR TITLE
Idea intellij plugin for gradle consistent versions

### PR DESCRIPTION
**Pull request by FinlayRJW**
This is a intellij plugin for managing versions.props files, see https://github.com/palantir/gradle-consistent-versions/pull/1234 for details

### CODE REVIEW

**Comment by FinlayRJW** [comment]
Looks like circle CI isn't running on this yet

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        classpath 'com.palantir.gradle.consistentversions:gradle-consistent-versions:2.25.0'
+        classpath 'com.palantir.baseline:gradle-baseline-java:5.67.0'
+        classpath 'com.palantir.gradle.gitversion:gradle-git-version:3.1.0'
+        classpath 'org.jetbrains.grammarkit:org.jetbrains.grammarkit.gradle.plugin:2022.3.2.2'
+    }
+}
+
+plugins {
+ id 'com.gradle.plugin-publish' version '1.3.0'
+}
```
Comment (CRogers) [inline build.gradle]: This can go.

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+apply plugin: 'com.palantir.external-publish'
+apply plugin: 'java-gradle-plugin'
+apply plugin: 'groovy'
+apply plugin: 'com.palantir.baseline'
+apply plugin: 'com.palantir.consistent-versions'
+apply plugin: 'com.palantir.git-version'
+apply plugin: 'com.palantir.jdks'
+apply plugin: 'com.palantir.external-publish-jar'
```
Comment (CRogers) [inline build.gradle]: This should only be on the subproject.

Comment (CRogers) [reply build.gradle]: Actually, not needed at all.

Comment (FinlayRJW) [reply build.gradle]: removed

File: build.gradle (hunk 1 above)
Comment (FinlayRJW) [reply build.gradle]: removed

**Comment by FinlayRJW** [comment]
Also think I have switched to junit5 by using [LightJavaCodeInsightFixtureTestCase5](https://github.com/JetBrains/intellij-community/tree/idea/242.22855.74/java/testFramework/src/com/intellij/testFramework/fixtures/LightJavaCodeInsightFixtureTestCase5.kt) for JUnit 5 (2021.1 and later) (https://plugins.jetbrains.com/docs/intellij/light-and-heavy-tests.html#-aoi4z9_24)

**Comment by CRogers** [comment]
👍🏻

**Pull request merged**